src/runtime_src/core/edge/drm/zocl/zocl_xclbin.c

```diff
@@ -593,6 +597,17 @@ zocl_xclbin_read_axlf(struct drm_zocl_dev *zdev, struct drm_zocl_axlf *axlf_obj)
 		ret = zocl_load_aie_only_pdi(zdev, axlf, xclbin);
 		if (ret)
 			goto out0;
+	} else if ( axlf_obj->za_flags == DRM_ZOCL_PLATFORM_FLAT && 
+		    axlf_head.m_header.m_mode == XCLBIN_FLAT && 
+		    axlf_head.m_header.m_mode != XCLBIN_HW_EMU &&
+		    axlf_head.m_header.m_mode != XCLBIN_HW_EMU_PR) {
+		/* 
+		 * Load full bitstream in case, it is enables in xrt runtime config
```

Comment:
it is enabled?

Reply:
not default enabled ... rephrased the comment